tsfc/driver.py

```diff
@@ -179,6 +179,8 @@ def facetarea():
         params = {}
         # Record per-integral parameters
         params.update(integral.metadata())
+        if params.get("quadrature_rule") == "default":
```

Comment:
I want to flag this up as a topic for further discussion. While this commit helps to satisfy both the Firedrake and the FFC test suite, the usage the `quadrature_rule` parameter in FEniCS/Firedrake is quite unclear. TSFC currently expects a `QuadratureRule` object here, while FFC seems to provide strings. (Maybe the `scheme` argument of `ffc.fiatinterface.create_quadrature`?) However, none is these usages seems to have tested in either Firedrake or FFC.